@ecomplus/storefront-app/src/components/js/EcOrderInfo.js

```diff
@@ -89,6 +95,7 @@ export default {
     i19shippingAddress: () => i18n(i19shippingAddress),
     i19transactionCode: () => i18n(i19transactionCode),
     i19ticketCode: () => i18n(i19ticketCode),
+    i19updatedCart: () => 'Carrinho atualizado',
```

Comment:
```suggestion
    i19cartUpdated: () => 'Carrinho atualizado',
```

Na verdade acho que seria **_cart updated_**.

> _updated cart_ remete a "o carrinho que foi atualizado" em vez de "o carrinho foi atualizado", deu pra entender o que eu quis explicar? haha

Reply:
Eu acho que o certo mesmo é updated cart, pois é como se fosse uma palavra composta. Agora o carrinho foi atualizado, ai sim é uma frase, dai cart seria o sujeito e atualizado seria um "adjetivo". Exemplo prático beautiful girl, adjetivo depois sujeito. Se fosse uma frase ai seria  girl was beautiful por exemplo. Eu acho né

Reply:
Pois é, falamos a mesma coisa :stuck_out_tongue: 
Mas no caso a frase é justamente no sentido "o carrinho foi atualizado", não? Não é a _palavra composta_, você está emitindo uma notificação dizendo após a atualização do carrinho.